src/components/pages/BottomBar.tsx

```diff
@@ -11,7 +11,7 @@ import { observer } from 'mobx-react'
 
 function BottomBar() {
     return (
-        <Container>
+        <Container style={{height: '55px' }}>
             <Stack direction="row" color="primary" justifyContent="center">
```

Comment:
@AlbertoCasasOrtiz Is there a way around specifying number of px in code, both here and in the expression on ModelViewPage? The current fix seems to work as of now but the view in dev tools shows the bar cropped on an iPad (though could be a chrome bug too). Could these numbers be moved to css? If not I'll merge this and see if it pops up later. on other devices. Thank you.